tutorials/evoked/plot_20_visualize_evoked.py

```diff
@@ -0,0 +1,247 @@
+"""
+.. _tut-visualize-evoked:
+
+Visualizing Evoked data
+=======================
+
+This tutorial shows the different visualization methods for
+:class:`~mne.Evoked` objects.
+
+.. contents:: Page contents
+   :local:
+   :depth: 2
+
+As usual we'll start by importing the modules we need:
+"""
+
+import os
+import numpy as np
+import mne
+from mayavi import mlab
```

Comment:
please don't use mayavi in the examples. We've tried to get rid of it everywhere with @GuillaumeFavelier